vividus-http-client/src/main/java/org/vividus/http/HttpDebug.java

```diff
@@ -0,0 +1,33 @@
+package org.vividus.http;
+
+import java.net.URI;
+
+import org.apache.http.client.methods.HttpRequestBase;
+
+public class HttpDebug extends HttpRequestBase
+{
+    public final static String METHOD_NAME = "DEBUG";
+
+    public HttpDebug()
+    {
+        super();
```

Comment:
why do you need explicit invocations of no-arg constructor?

Reply:
to have the ability to set `URI` via setter

Reply:
I was talking about super() invocations, because no-args constructors are invoked in java explicitly before all instructions if we don't specify other constructor